lms/templates/discussion/_discussion_inline.html

```diff
@@ -13,6 +13,15 @@
     data-user-create-comment="${json_dumps(can_create_comment)}"
     data-user-create-subcomment="${json_dumps(can_create_subcomment)}"
     data-read-only="${'false' if can_create_thread else 'true'}">
+    % if not user.is_authenticated():
```

Comment:
The `request` context object is added by Django. I'm not certain that happens in this case, since we are rendering a fragment. The `user` is explicitly added to the context, so I am using it: https://github.com/edx/edx-platform/blob/978e9fe656bb249237ba1ceea0ebec1aa0b06fa7/openedx/core/lib/xblock_builtin/xblock_discussion/xblock_discussion.py#L173.